cpt-repo.tex

```diff
@@ -21,6 +21,58 @@ \chapter{Creating a Repository (Stratum~0)}
 In order to provide a writable \cvmfs\ repository, \cvmfs\ uses a union file system that combines a read-only \cvmfs\ mount point with a writable scratch area~\cite{unionfs04,aufs}.
 Figure~\ref{fig:installwebserver} outlines the process of publishing a repository.
 
+\pagebreak
+
+\section{\cvmfs\ Server Quick-Start Guide}
+\label{sct:quickstart}
+
+\subsection{System Requirements}
+\begin{itemize}
+\item Apache HTTP server \emph{OR} S3 compatible storage service
+\item AUFS union file system in the kernel
+\item Officially supported platforms
+\begin{itemize}
+    \item Scientific Linux 5 (64 bit)
+    \item Scientific Linux 6 (64 bit - with custom AUFS enabled kernel)
```

Comment:
Here I think the difficult part is how to get the custom kernel (install cvmfs-release package, steer yum to fetch kernel from cernvm-kernel repo, lock it there).  That should be explained.